### ISSUE TITLE
List of dependents sorted by download count

### ISSUE THREAD

**Issue by vjeux** [yarnpkg/website#739]
I'm trying to figure out where the almost 2 million downloads for prettier are coming from. Would be nice to have a list of all the dependents sorted by number of downloads.

![24852251_216477648894393_370888951655792518_n](https://user-images.githubusercontent.com/197597/33800031-ed1063be-dcec-11e7-8820-20409899afef.jpg)

cc @Haroenv

**Comment by Haroenv** [comment]
That’s actually a big we have in where the number comes from. Npm doesn’t really have a good api for getting it, it stopped working correctly some time ago. The link there leads to the list though.  See also https://github.com/algolia/npm-search/issues/63

**Comment by vjeux** [comment]
Yeah, npm info is also not useful so I figured I would ask you :p github also has a list of dependents but also not sorted by stars :(

I guess I’ll continue to live in ignorance for now :p

**Comment by Haroenv** [comment]
I’ll take a look what a good source can be though
